tutorials/compare-transpiler-settings/compare-transpiler-settings.ipynb

```diff
@@ -0,0 +1,417 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "a706ab82-02f1-41ab-9400-1b0d3d887547",
+   "metadata": {},
+   "source": [
+    "# Compare transpiler settings\n",
+    "\n",
+    "## Background\n",
+    "\n",
+    "To ensure faster and more efficient results, as of 1 March 2024, circuits and observables need to be transformed to only use instructions supported by the system (referred to as instruction set architecture (ISA) circuits and observables) before being submitted to the Qiskit Runtime primitives.  One common way to do this is to use the transpiler's `generate_preset_pass_manager` function.  However, you might choose to follow a more manual process.\n",
```

Comment:
```suggestion
    "To ensure faster and more efficient results, as of 1 March 2024, circuits and observables need to be transformed to only use instructions supported by the system before being submitted to the Qiskit Runtime primitives. We call these _instruction set architecture_ (ISA) circuits and observables. One common way to do this is to use the transpiler's `generate_preset_pass_manager` function.  However, you might choose to follow a more manual process.\n",
```

Just to avoid nested parentheses.